torch/quantization/fuse_modules.py

```diff
@@ -111,6 +144,5 @@ def fuse_modules(model, modules_to_fuse):
             >>> output = m(input)
 
     """
-    named_module_dict = {name: mod for name, mod in model.named_modules()}
     for module_list in modules_to_fuse:
-        _fuse_modules(model, named_module_dict, module_list)
+        _fuse_modules(model, module_list)
```

Comment:
can we add a check that if passed in thing is a list of strings just do it? (and update the doc accordingly)

```
if len(modules_to_fuse) > 0 and isinstance(modules_to_fuse, str):
  _fuse_modules(model, modules_to_fuse)
else:
  for module_list ...
```

Reply:
What is passed in is a list of list of strings, like, for example:
```python
        fuse_modules(model, [['conv1', 'relu1'] ,
                             ['features.0.0', 'features.0.1', 'features.0.2'],
                             ['features.1.0', 'features.1.1', 'features.1.2'],
                             ['features.2.0', 'features.2.1', 'features.2.2'],
                             ['classifier.0', 'classifier.1']])
```
So, not sure what you mean here? can you elaborate?

Reply:
I mean also supporting passing a single list of strings (in addition to list of list of strings). It's just convenient to also accept it without extra nesting:

`fuse_modules(model, ['conv1', 'relu1'])`